src/size-rhythms.md

```diff
@@ -0,0 +1,65 @@
+
+**Note** write `1px` not `0,0625rem`!
+
+Anything that should get rendered in `1px` on a screen (such as a border) should be written in pixels. It's very important not to write it in its rem equivalent (`0,0625rem`). This is because depending on the physical pixel density of screens, a `0,0625rem` thick border for example, may get rendered like a semi-transparent line; or even worse, never get rendered in some situations.
+
+## Exceptions in sizing rhythms
+There are exceptions where we do not have to follow the abovementioned sizing convention (multiples of 4). These exceptions are:
+* Numbers `1` and `2` are allowed to use when defining some visual details such as borders.
+* `font-size`: how fonts are rendered depend on the typography and design of the font itself. Fonts do not follow the same logic of simple geometric shapes of the UI. Therefore, using incremental rhythm based on 4 for font sizes (e.g. for H1, H2, H3, etc...) may generate huge font sizes which won't be usable. Also other numbers such as 14 or 13 may appear more appealing for a balanced and easily readable text, compared to 16, or 12.
+* `border-radius`: how a value of 4 is rendered as a border radius is slightly too-little for the type of visual expression we want to achieve for example on a small clickable element like a button. This is while a value of 8 may be too much. Thus, the designer may pick an arbitrary value that helps the visual expression.
+* `box-shadow`, `drop-shadow`, `blur`: How far a shadow reaches until it's faded out, or how blurry something looks should also be flexible depending on the visual expression that you want to achieve.
+
+## Be believer, but not a fanatic!
+This is about having a consistent rhythm in the UI, not about religiously following hard guidelines. In some cases, you will find yourself in situations in which some small detail in the UI is not following the 4-units rhythm, while everything else does.
+
+Let's look at the example below:
+
+<limel-example-size-edge-case></limel-example-size-edge-case>
+
+Here you see that the distance between the picture and top, bottom, and left of the rows is 6 units. However, the `line-height` or `height` of the picture itself (and other details) are multiplies of 4. Now this is a fantasy example for illustrative purposes, but cases like this may happen in real life when designing components and layouts. For example you may choose to use 6 units for distance on a certain type of screen to achieve a higher visual density.
+
+So, if you can justify that 6 works fine for your design, and rest of your structure is following the 4x rhythm; then go for it! In this example, the accumulated height of the row (which is more decisive for the general rhythm) will be 24 (4 * 6). So you don't "have to" change 6 to 4 or 8 so that all numbers become multiplies of 4.
```

Comment:
```suggestion
So, if you can justify that 6 works fine for your design, and the rest of your structure is following the 4x rhythm; then go for it! In this example, the accumulated height of the row (which is more decisive for the general rhythm) will be 24 (4 * 6). So you don't "have to" change 6 to 4 or 8 so that all numbers become multiplies of 4.
```